src/clusterfuzz/_internal/base/tasks.py

```diff
@@ -97,6 +100,9 @@ def __init__(self, task):
 
 def queue_suffix_for_platform(platform):
   """Get the queue suffix for a platform."""
+  if SUBQUEUE_IDENTIFIER in platform:
+    return '-' + platform.lower().replace(':', '-')
```

Comment:
It should be harmless to always do this replacement right? 

We can just chain the replace without the if and add a comment explaining the cases.

Reply:
Done.